### DISCUSSION TITLE
Vertical Electrical Sounding

### DISCUSSION THREAD

**Discussion by dewitrilestari123** [gimli-org/pyGIMLi#753] [Q&A] [answered]
Hello, I am currently a beginner user of PyGIMLi, and I am tasked with performing inversion modeling of vertical electrical sounding using PyGIMLi. However, I am encountering difficulties in extracting the resistivity value of each layer. I am aiming to generate a resistivity log vs depth output similar to the one produced by Software Progress (I dropped the example below). I would like to inquire if PyGIMLi is capable of creating a 1D model of each layer and displaying the resistivity values akin to Software Progress. If this is possible, could you kindly provide me with the corresponding code?

![IMG_20240719_101304](https://github.com/user-attachments/assets/38a8a43f-9c77-4100-abcb-e8bade50c600)

**Comment by halbmy** [comment, marked as answer]
There is no plotting function like in the image where the values are printed, but it should be very easy to generate using matplotlib. pyGIMLi just has resistivity-depth curves or coloured depth columns.

For the modelling, you just need the thickness and resistivity values as vectors. Please have a look at the VES examples.
